docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx

```diff
@@ -31,7 +31,9 @@ In this chapter, you will find guidance to set up the WG1000 Gateway.
 
 1. The station must be fully assembled (parts and batteries).
 1. A working smartphone with connection to the internet.
-3. A reliable 2.4 GHz WiFi network (5 GHz is not supported) for the WG1000 Gateway to connect to.
+1. A reliable 2.4 GHz WiFi network (5 GHz is not supported) for the WG1000 Gateway to connect to.
+1. The WeatherXM app installed on your smartphone. [WeatherXM Application](/wxm-devices/D1-bundle/D1-connect-and-claim#download-weatherxm-app-and-login)
```

Comment:
Wrong link, it goes to D1 not the M5's one! Also, it doesn't make sense as it is now that a "WeatherXM Application" is after the dot. Please make it more human-friendly.